ivy/functional/frontends/jax/numpy/mathematical_functions.py

```diff
@@ -623,6 +622,28 @@ def polyint(p, m=1, k=None):
     return ivy.divide(ivy.concat((p, k_arr)), coeff).astype(p.dtype)
 
 
+@with_unsupported_dtypes(
+    {"0.3.14 and below": ("float16",)},
+    "jax",
+)
+@to_ivy_arrays_and_back
+def polydiv(u, v, *, trim_leading_zeros=False):
+    u, v_arr = ivy.promote_types_of_inputs(u, v)
+    n = v_arr.shape[0] - 1
+    m = u.shape[0] - 1
+    scale = 1. / v_arr[0]
+    q = ivy.zeros((max(m - n + 1, 1),), dtype=u.dtype)
+    r = ivy.copy_array(u)
+    for k in range(0, m - n + 1):
+        d = scale * r[k]
+        q[k] = d
+        r[k:k+n+1] -= (d * v_arr)
+    if trim_leading_zeros:
+        r = trim_zeros(r, trim='f')
+        # TODO: Need to find a way to control tolerance of trim_zeros
```

Comment:
You could write a new function to allow zero trimming with tolerance, either as a helper function here, or in our experimental api (I have added `ivy.trim_zeros` to our list). Otherwise you could leave this argument unimplemented and add a ToDo comment explaining why.